README.md

```diff
@@ -47,18 +47,16 @@ coverage:
   steps:
     - name: Install Go
       if: success()
-      uses: actions/setup-go@v2-beta
+      uses: actions/setup-go@v2
+      with:
+        go-version: 1.15.x
```

Comment:
why not use setup-go automate version select?

Reply:
i prefer explicit versioning for reproducable builds

Reply:
in for user of copy code, not is good idea.